code-challenges.test.js

```diff
@@ -28,42 +33,172 @@ const numbersArray2 = [24, 27, 30, 33, 36]
+//   ✕ returns an array with all the words capitalized (2 ms)
+
+// ● capitalizedArray › returns an array with all the words capitalized
+
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
+//   Received: undefined
+
+//      98 | describe("capitalizedArray", () => {
+//      99 |     it("returns an array with all the words capitalized", () => {
+//   > 100 |         expect(randomNouns1.map.toUpperCase).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+//         |                                              ^
+//     101 |         expect(randomNouns2.map.toUpperCase).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+//     102 |
+//     103 |     })
+
+//     at Object.<anonymous> (week-2-assessment-VMMarin/code-challenges.test.js:100:46)
+
+
+
+// b) Create the function that makes the test pass.
+
 const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+//Pseudocode
+// Create a function that takes in an array of words and returns an array with all the words capitalized.
+// create a function that takes in an array and returns each word capitalized
+// declare a function that takes in an array
+// create a local variable that maps through the array
+// with each value access the first index and capitalize the letter then concatenate the remainder of the word
+// return the local variable joined into a string
 
-// b) Create the function that makes the test pass.
+const capitalNouns = (array) => {
+  let eachItem = array.map(value => {
+    return value[0].toUpperCase() + value.substring(1)
+  })
+   return eachItem.join(", ")
+}
 
+console.log(capitalNouns(randomNouns1))
+console.log(capitalNouns(randomNouns2))
 
+//
 
 // --------------------3) Create a function that takes in a string and logs the index of the first vowel.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+
+describe("indexeddArray", () => {
+    it("logs the index of the first vowel.", () => {
+        expect().toEqual(
+    })
+})
+
+
+
+
+
 const vowelTester1 = "learn"
 // Expected output: 1
-const vowelTester2 = "academy"
+//const vowelTester2 = "academy"
 // Expected output: 0
-const vowelTester3 = "challenges"
+//const vowelTester3 = "challenges"
 // Expected output: 2
 
 
 // b) Create the function that makes the test pass.
+
+//input will be string
+///output will be index of first vowel of string
+//utilizing string.search(vowels) to identify vowels
+
+//Had a lot of trouble with this one - roughly 2 hours of creating and erasing code, googling, etc. My mind tells me it should be very simple, but every output was an error. 
+
```

Comment:
If you are stuck for more than 30 min you should reach out in your cohort Slack channel!